addons/account_edi_ubl_cii/models/res_partner.py

```diff
@@ -133,49 +133,81 @@ def _check_peppol_fields(self):
                 if error:
                     raise ValidationError(error)
 
-    @api.depends('country_code')
+    def _filter_be_partners(self, partner):
```

Comment:
put `_deduce_peppol_country_code` as `_deduce_country_code` directly in account? Then, you can remove this method I think

Reply:
but we only need to deduce it for ubl_cii module :thinking: